core/templates/css/oppia.css

```
@@ -4647,8 +4655,23 @@ div.oppia-issues-learner-action {
   width: 100%;
 }
 
+/* Increase no. of columns for cards on contributors profile page */
```

I still do not understand how this comment relates to the CSS below.

---

Yeah, so the css below is to increase the no. of columns of cards in the page, as you can see in the video, there are 3 cards showing up, but originally there was only 1 column and all the cards were in that column. so the `grid-template-column: max-content max-content max-content` is for that

---

Okay, I get it now. Lets phrase the comment differently though 
```suggestion
/* By using 'grid-template-columns' we specify the number of columns for different display sizes. */
```